Formula/orientdb.rb

```diff
@@ -48,23 +43,36 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "orientdb"
+    system "#{bin}/orientdb", "stop"
+    system "#{bin}/orientdb", "start"
+    sleep 3
+
+    ensure
```

Comment:
`ensure`  is at the method level so should be indented back two spaces and the `end` following it removed.